Web Scrapper Bot/Readme.md

```diff
@@ -0,0 +1,9 @@
+# This is a Web Scrapping Bot
+
+# Language, Libraries and Modules used-
```

Comment:
Please add a requirements.txt file.
You can use `pip freeze > requirements.txt` to generate requirement file.
And we can remove this from readme after that